data/static.yaml

```diff
@@ -18,7 +18,7 @@ pe_status_check::checks:
   S0017: "Determines if there are any OutOfMemory errors in the Puppetdb JVM"
   S0018: "Determines if there are any OutOfMemory errors in the Orchestrator JVM"
   S0019: "Determines if there are sufficient jrubies available to serve agents"
-  S0020: "Determines"
+  S0020: "Determines if the status endpoint for puppet server returns for port 4433"
```

Comment:
This will be for the console-services api end point, and it returns true if all the services report they are all good